### DISCUSSION TITLE
Integration of 10x PolyA Data with Fixed RNA Profiling

### DISCUSSION THREAD

**Discussion by ckfromCN** [theislab/scvelo#1149] [Q&A]
I am currently working on a project where I am integrating 10x Genomics PolyA-enriched RNA sequencing data with Fixed RNA Profiling data (with bbknn) to quantify macrophage and monocyte populations. Given that the Fixed RNA Profiling is based on probes that are only 14% spliced, I have concerns regarding the impact this might have on the results of my analysis. #267

**Comment by aigerim2398** [comment]
Hi @ckfromCN 
I am currently trying to do the same thing with my datatsets. Did you manage to find a solution?
